src/components/TagWall/_index.scss

```diff
@@ -4,12 +4,14 @@
 /// @copyright IBM Security 2019
 ////
 
+@import '../../../globals/variables/index';
```

Comment:
- [ ] This should resolve as `@import '../../globals/variables/index';` otherwise the CSS build breaks

Reply:
🤤 Well spotted - thanks!